projects/ngx-charts-on-fhir/src/lib/fhir-chart-summary/fhir-chart-summary/fhir-chart-summary.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="chart-summary">
   <ng-container *ngIf="configService.timelineRange$ | async as range">
-    <fhir-chart-summary-card
-      *ngFor="let layer of layerManager.selectedLayers$ | async"
-      [color]="colorService.getColorGradient(layer)"
-      [title]="layer.name"
-      icon="info"
-      class="dataset-summary mat-elevation-z2"
-    >
-      <dynamic-table [data]="summarize(layer, range)"></dynamic-table>
-    </fhir-chart-summary-card>
+    <div class="summary-card" *ngFor="let layer of layerManager.selectedLayers$ | async">
+      <fhir-chart-summary-card  *ngIf="layer.enabled"
```

Comment:
This leaves an empty div for each hidden layer. It's barely noticeable with one hidden layer, but with more layers it can become a large gap.
![image](https://user-images.githubusercontent.com/2105775/216080700-d9dfae59-5ee6-474d-9447-c2dc6c6bc71b.png)

Reply:
You can fix this either by using `<ng-container>` instead of `<div>`
or by creating an observable like:
```
  enabledLayers$ = this.selectedLayers$.pipe(map((layers) => layers.filter((layer) => layer.enabled)));
```
which might be useful in other places too.